scripts/aws/cfn/microservice.yml

```diff
@@ -236,6 +236,10 @@ Parameters:
   PaperChannelCxId:
     Type: String
     Description: 'PaperChannel cx-id'
+  
+  ExternalChannelCxId:
+    Type: String
+    Description: 'ExternalChannel cx-id'
```

Comment:
@alessioangelotti-pagopa cambierei questa descrizione "CxId used by service desk for requests towards External Channel"